### DISCUSSION TITLE
Welcome our public Slack Workspace!

### DISCUSSION THREAD

**Discussion by dario-vega** [oracle/nosql#49] [Announcements]
Want to keep in touch? Maybe via Slack? Click the [link ](http://bit.ly/odevrel-slack) to join our public Slack Group!
- join the channel "oracle-nosql" to learn more about the Oracle NoSQL database
- If you're looking for a HOL or webinar, please join the specific HOL channel

**Comment by hsiong** [comment]
Hi , "Slack: This link is no longer active", could u give a new link ?

**Reply by dario-vega** [reply]
Bizarre, it is working for me. Let us try this one : https://join.slack.com/t/oracledevrel/shared_invite/zt-1flqpwig2-iQLR~DHT4JYa_mv9Am9DPg

I will investigate what is happening

**Reply by dario-vega** [reply]
Okay, For security, invite links can be used by up to 100 people and can be set to expire.

the link does periodically expire but you can always find the latest link at the bottom of https://developer.oracle.com/

![image](https://user-images.githubusercontent.com/66302636/188732100-3848353c-f5fc-4ce6-ab5c-ffebd1cc6859.png)

**Reply by hsiong** [reply]
Yep, got it .   Thank you
